### PR TITLE
Re-create glide files for RC4

### PR DESCRIPTION
**Pull request by robbrockbank**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,29 +1,40 @@
 package: github.com/projectcalico/libcalico-go
-homepage: https://projectcalico.org
-license: Apache-2.0
```
Comment (neiljerram) [inline glide.yaml]: Does it matter that the homepage and license have been lost here?

File: glide.yaml [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
-owners:
-- name: Rob Brockbank
-  email: rob@tigera.io
```
Comment (neiljerram) [inline glide.yaml]: And the owner!

File: glide.yaml (hunk 1 above)
Comment (tomdee) [inline glide.yaml]: No - none of our other glide files have them

File: glide.yaml (hunk 2 above)
Comment (tomdee) [inline glide.yaml]: As above :smile:

**Pull request merged**